miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -3048,6 +3097,10 @@ public static InstrumentDto asDto(@Nonnull Instrument from) {
     setString(dto::setIdentificationBarcode, from.getIdentificationBarcode());
     setId(dto::setWorkstationId, from.getWorkstation());
     setString(dto::setWorkstationAlias, maybeGetProperty(from.getWorkstation(), Workstation::getAlias));
+    if (from.getServiceRecords() != null) {
+
+      dto.setServiceRecords(from.getServiceRecords().stream().map(Dtos::asDto).collect(Collectors.toSet()));
+    }
```

Comment:
Please also add `setServiceRecords()` in the `public static Instrument to(@Nonnull InstrumentDto dto)` function below.